edx_proctoring/tests/test_email.py

```diff
@@ -31,22 +31,25 @@ class ProctoredExamEmailTests(ProctoredExamTestCase):
     All tests for proctored exam emails.
     """
 
-    def setUp(self):
-        """
-        Build out test harnessing
-        """
-        super(ProctoredExamEmailTests, self).setUp()
-
-        # Messages for get_student_view
-        self.proctored_exam_email_subject = 'Proctoring Session Results Update'
-        self.proctored_exam_email_body = 'the status of your proctoring session review'
-
     @ddt.data(
-        ProctoredExamStudentAttemptStatus.submitted,
-        ProctoredExamStudentAttemptStatus.verified,
-        ProctoredExamStudentAttemptStatus.rejected
+        [
+            ProctoredExamStudentAttemptStatus.submitted,
+            'Proctoring Review In Progress',
+            'was submitted successfully',
+        ],
+        [
+            ProctoredExamStudentAttemptStatus.verified,
+            'Proctoring Results',
+            'was reviewed and you met all exam requirements',
+        ],
+        [
+            ProctoredExamStudentAttemptStatus.rejected,
+            'Proctoring Results',
+            'the team found one or more violations',
+        ]
     )
-    def test_send_email(self, status):
+    @ddt.unpack
+    def test_send_email(self, status, expected_subject, expected_message_string):
```

Comment:
While you are in here, Is it possible to consolidate `test_email_not_sent` and `test_not_send_email`?